app/next-client-app/components/concepts/ConceptTagsV3.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useOptimistic } from "react";
+import { deleteConceptV3 } from "@/api/concepts";
+import { Button } from "@/components/ui/button";
+import { ApiError } from "@/lib/api/error";
+import { Cross2Icon } from "@radix-ui/react-icons";
+import { toast } from "sonner";
+import { ConceptDetailsSheet } from "./ConceptDetailsSheet";
+
+export function ConceptTagsV3({
+  concepts,
+  scanReportId,
+  tableId,
+  fieldId,
+  valueId,
+}: {
+  concepts: ScanReportConceptV3[];
+  scanReportId: string;
+  tableId: string;
+  fieldId: number;
+  valueId: number;
+}) {
+  const [optimisticConcepts, setOptimisticConcepts] = useOptimistic(
+    concepts,
+    (state, conceptIdToDelete: number) =>
+      state.filter((concept) => concept.id !== conceptIdToDelete)
+  );
+
+  const handleDelete = async (conceptId: number) => {
+    try {
+      setOptimisticConcepts(conceptId);
+      await deleteConceptV3(
+        conceptId,
+        `/scanreports/${scanReportId}/tables/${tableId}/fields/${fieldId}/beta`
+      );
+      toast.success("Concept Id Deleted");
+    } catch (error) {
+      const errorObj = JSON.parse((error as ApiError).message);
+      toast.error(
+        `Unable to delete Concept id from value Error: ${errorObj.detail}`
+      );
+      console.error(error);
+    }
+  };
+
+  return optimisticConcepts && optimisticConcepts.length > 0 ? (
+    <div className="flex flex-col items-start w-[250px]">
```

Comment:
```suggestion
    <div className="flex flex-col items-start">
```
I don't think you need `w` here, cause it is already determined by `w` in line 58.

Can you also do me a favour? Please can you modify the `max-w` to 300px in the concept tag v1 to be consistent? 🙏